app/lib/clients/twitter_client.rb

```diff
@@ -11,11 +11,26 @@ def self.authenticate
     client
   end
 
+  TwitterQuery = Struct.new(:created_at, :text, :favorite_count, :retweet_count, :reply_count)
+
+  def self.reply_count(id, client)
+    replies = client.search("to:mojotech ", since_id: id, include_entities: true)
+    .select { |res| res.in_reply_to_status_id == id }
```

Comment:
This indentation is off. I'd also expect `.search(....)` to be on a new line as well. Is rubocop not enforcing this?